tests/matmul/generate_e2e_matmul_tests.py

```diff
@@ -102,14 +102,31 @@ def get_test_shapes(shapes_id: ShapesId):
     # 2. Some shapes are commented out: they used to be tested but have been
     #    disabled to improve the trade-off between test coverage and build
     #    latency.
+
     if shapes_id == ShapesId.SMALL:
+        return [
+            # some "nice" multiple of 8 shapes
+            TestShape(m=8, k=16, n=32, accumulate=False),
+            TestShape(m=16, k=8, n=16, accumulate=False),
+            # some arbitory shapes
+            TestShape(m=9, k=15, n=7, accumulate=False),
+            TestShape(m=52, k=63, n=52, accumulate=False),            
```

Comment:
Do these shape really work? Very nice if so! Maybe not for bfloat16 yet though?

Reply:
Yes, I would expect them to work for scalar path, which doesnt exist for bf16.